src/main/java/org/desarrolladorslp/technovation/models/User.java

```diff
@@ -30,6 +31,7 @@
     private static final long serialVersionUID = 6589898843278314546L;
 
     @Id
+    @Type(type="pg-uuid")
```

Comment:
Can you try removing this? I think with the last approach this is not needed anymore